README.md

```diff
@@ -13,7 +13,19 @@ Installing `gls` on your machine is pretty simple: just clone the repo and run `
 ```bash
 $ git clone github.com/ozansz/gls
 $ cd gls
-$ go run cmd/gls.go
+$ go build cmd/gls.go
+```
+
+After you run `go build command`, an executable file name `gls` is created. You can modify `.bashrc`, `.zshrc`, or another 
```

Comment:
we can change “`go build command`” to “`go build` command”

Reply:
Done in [c7ea452](https://github.com/ozansz/gls/pull/3/commits/c7ea452552076f232b1b29d99c8e3c6cd7097ef9).